hackathon_site/hackathon_site/tests.py

```diff
@@ -62,73 +77,134 @@ def _get_random_email():
             email = f"{uuid[:10]}@{uuid[10:20]}.com"
         return email
 
-    def _make_full_registration_team(self, team=None, self_users=True):
+    def _make_full_registration_team(self, team=None, self_users=True, num_users=4):
         if team is None:
             team = RegistrationTeam.objects.create()
 
         if self_users:
+
             user1 = self.user
+            cumulative_users = [user1]
+            if num_users > 1:
```

Comment:
You can come up with a better way to write this using a for loop.

I don't care about using the painter's names either (unless we use them elsewhere in tests), you can just generate random values for all the fields.

Reply:
fixed